docs/HISTORY.txt

```diff
@@ -5,7 +5,8 @@ Changelog
 1.5.3 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Fix iframe auto resizer [Nachtalb]
```

Comment:
Could you maybe elaborate on what was wrong?

Reply:
Would something like that be sufficient?
```
- Fix iframe auto resizer not being triggered on plone 4 [Nachtalb]
- Add missing auto resizer dependency on plone 5 [Nachtalb]
```

Reply:
Would you mind putting this?
Ensure iframeResizer.contentWindow is included on Plone 5 - in case iframe includes a page from the the same site